scripts/chaos_eq_report.py

```
@@ -0,0 +1,297 @@
+        "- Keep normal equipment portable at the item-data level and apply runtime `can_char_use_item()` and body-slot checks before putting an object in the bag.",
+        "- Treat a missing/unusable required object as a fail-closed kit failure; only expected race/body-slot omissions are skipped.",
+        "- Keep consumables inside the bag, with bounded starter quantities based on high-level carried/container usage rather than copying observed stockpiles.",
+        "- Treat Bard instruments, the master spellbook, and the Shaman three-sphere totem as explicit fundamentals instead of accidentally filtering them as ordinary gear.",
+        "",
+        "## Evidence snapshot",
+        "",
+        f"- Cohort: active characters at level >= {cohort.get('threshold', 50)}.",
+        f"- Characters: {cohort.get('characters', 0)}; level 51+: {cohort.get('level_51_or_higher', 0)}; level 56+: {cohort.get('level_56_or_higher', 0)}; observed maximum: {cohort.get('max_level', 0)}.",
+        f"- Static object sources: {analysis.get('static_reconciliation', {}).get('area_files', 0)} active area files and {analysis.get('static_reconciliation', {}).get('area_records', 0)} parsed object records.",
+        f"- Candidate templates after database/static reconciliation: {len(analysis.get('candidates', []))}; generated class profiles: {len(catalog.get('profiles', {}).get('standard', {}))} standard and {len(catalog.get('profiles', {}).get('enhanceable', {}))} enhanceable.",
+        f"- Archive inventory: `player_data` 785 rows; `player_items` 50,239 rows; `player_item_affects` 34,008 rows; `player_item_extra_descr` 289,015 rows; `wiki_objects` 19,661 rows; `classes` 31; `races` 101; all item rows were reachable through `container_id -> player_items.id`, with maximum observed nesting depth 3.",
```

Ruff (0.16.3)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)